tests/Unit/PublicPropertyManagerTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace Tests\Unit;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Str;
+use Livewire\Exceptions\CannotRegisterPublicPropertyWithoutExtendingThePropertyHandlerException;
+use Livewire\LivewireProperty;
+use Livewire\Livewire;
+
+class PublicPropertyManagerTest extends TestCase
```

Comment:
I think another test that's needed is ensuring that a property of:
```php
public ?Foo $foo = null;
```
can receive `null` correctly. `ReflectionProperty` has a `allowsNull()` method I think.

Reply:
@jameslkingsley yeah agreed, @jonassiewertsen can you add a test for the above as requested.